### PR TITLE
Support for importing sample-file-scoped chromatograms from SKYD

### PR DESCRIPTION
**Pull request by labkey-jeckels**
and rendering them on a new sample file details page

### CODE REVIEW

File: test/src/org/labkey/test/tests/targetedms/TargetedMSSampleFileChromInfoTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+        Connection cn = createDefaultConnection(false);
+        SelectRowsCommand cmd = new SelectRowsCommand("targetedms", "samplefilechrominfo");
+        cmd.setRequiredVersion(9.1);
+        cmd.setColumns(Arrays.asList("SampleFileId", "StartTime", "EndTime", "TextId", "NumPoints", "UncompressedSize", "ChromatogramFormat", "ChromatogramOffset", "ChromatogramLength", "SampleFileId/ReplicateId"));
+
+        Assert.assertEquals("Wrong number of SampleFileChromInfo rows", 8, cmd.execute(cn, "/" + getProjectName()).getRowset().getSize());
+
+        cmd.setFilters(Arrays.asList(new Filter("StartTime", 0.0), new Filter("EndTime", 79.966)));
+        Assert.assertEquals("Wrong number of time-filtered SampleFileChromInfo rows", 8, cmd.execute(cn, "/" + getProjectName()).getRowset().getSize());
```
Comment (labkey-ankurj) [inline test/src/org/labkey/test/tests/targetedms/TargetedMSSampleFileChromInfoTest.java]: Test fails after apply end time filter. It's the same behavior in grid view as well when applying end time filter.

Comment (labkey-jeckels) [reply test/src/org/labkey/test/tests/targetedms/TargetedMSSampleFileChromInfoTest.java]: Earlier today I merged a fix for this in the platform repo:

https://github.com/LabKey/platform/pull/698

Comment (labkey-jeckels) [reply test/src/org/labkey/test/tests/targetedms/TargetedMSSampleFileChromInfoTest.java]: TeamCity is passing

https://teamcity.labkey.org/viewLog.html?buildId=878421&buildTypeId=LabkeyTrunk_ModuleSuites_TargettedMSPostgres&tab=testsInfo&branch_LabkeyTrunk_ModuleSuites=sampleFileChromatograms

Comment (labkey-ankurj) [reply test/src/org/labkey/test/tests/targetedms/TargetedMSSampleFileChromInfoTest.java]: thanks, it passed.

**Pull request merged**